src/azure-cli/azure/cli/command_modules/vm/_image_builder.py

```diff
@@ -370,16 +376,39 @@ def process_img_tmpl_output_add_namespace(cmd, namespace):
 
 # region Custom Commands
 
-def create_image_template(  # pylint: disable=too-many-locals
+def create_image_template(  # pylint: disable=too-many-locals, too-many-branches, too-many-statements
         cmd, client, resource_group_name, image_template_name, location=None,
         source_dict=None, scripts_list=None, destinations_lists=None, build_timeout=None, tags=None,
         source=None, scripts=None, checksum=None, managed_image_destinations=None,  # pylint: disable=unused-argument
-        shared_image_destinations=None, no_wait=False):  # pylint: disable=unused-argument, too-many-locals
+        shared_image_destinations=None, no_wait=False, image_template=None):  # pylint: disable=unused-argument, too-many-locals
     from azure.mgmt.imagebuilder.models import (ImageTemplate, ImageTemplateSharedImageVersionSource,
                                                 ImageTemplatePlatformImageSource, ImageTemplateIsoSource, ImageTemplateManagedImageSource,  # pylint: disable=line-too-long
                                                 ImageTemplateShellCustomizer, ImageTemplatePowerShellCustomizer,
                                                 ImageTemplateManagedImageDistributor, ImageTemplateSharedImageDistributor)  # pylint: disable=line-too-long
 
+    if image_template is not None:
+        if os.path.exists(image_template):
+            with open(image_template) as f:
+                content = f.read()
```

Comment:
compare to URL logic, seems it should be `content = json.loads(f.read())` here ?